Exercise_F.js

```diff
@@ -10,6 +10,14 @@
 
 function occurrence(string, targetChar) {
   // CODE HERE
+  var x = string.toLowerCase();
+  var count = 0;
+  for (i=0; i<string.length;i++){
+    if(targetChar == string[i]){
+      count++;
+    }
+  }
+  return count
 }
 

```

Comment:
Nice work here very good logic
If you have some time you could come back and see if you're able to make the code cleaner still.